Model/Ui/CapabilitiesConfigProvider.php

```diff
@@ -0,0 +1,46 @@
+<?php
+namespace Omise\Payment\Model\Ui;
+
+use Magento\Checkout\Model\ConfigProviderInterface;
+use Magento\Payment\Api\PaymentMethodListInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Omise\Payment\Model\Capabilities;
+use Omise\Payment\Model\Config\Installment as OmiseInstallmentConfig;
+
+class CapabilitiesConfigProvider implements ConfigProviderInterface
+{
+    private $_storeManager;
+
+    /**
+     * @var Magento\Payment\Api\PaymentMethodListInterface;
+     */
+    private $_paymentLists;
+
+    public function __construct(
+        Capabilities               $capabilities,
+        PaymentMethodListInterface $paymentLists,
+        StoreManagerInterface      $storeManager
+    ) {
+        $this->capabilities    = $capabilities;
+        $this->_paymentLists   = $paymentLists;
+        $this->_storeManager   = $storeManager;
+    }
+
+    /**
+     * Retrieve assoc array of checkout configuration
+     *
+     * @return array
+     */
+    public function getConfig()
+    {
+        $listOfActivePaymentMethods = $this->_paymentLists->getActiveList($this->_storeManager->getStore()->getId());
+        foreach ($listOfActivePaymentMethods as $method) {
+            if ($method->getCode() === OmiseInstallmentConfig::CODE) {
+                return [ 
+                    'capabilities' => $this->capabilities->retrieveInstallmentBackends()
```

Comment:
@jacstn I would like to suggest to rename this one to something else as well (`installment_backends`, `installment_capabilities` would make it).
The reason is that `window.checkoutConfig` variable in Javascript file is sharing across the checkout page.
Meaning that if in the future you want to retrieve a list of banks that accept Internet Banking then it will come back to the naming issue.